### PR TITLE
API reference docs for RTD

### PR DESCRIPTION
**Pull request by jamadden**
Currently based on #6.

### CODE REVIEW

**Comment by jamadden** [comment]
Viewable at http://zopeapponlinehelp.readthedocs.io/en/docs/

File: src/zope/app/onlinehelp/browser/__init__.py [hunk 1]
```diff
@@ -50,6 +50,7 @@ def renderTopic(self):
 
 
 class ContextHelpView(BrowserView):
+    "Contextual help view."
```
Comment (mgedmin) [inline src/zope/app/onlinehelp/browser/__init__.py]: Perhaps use tripple quotes so it looks like a docstring?

**Review by mgedmin — APPROVED** [review]
The last commit LGTM; I didn't review the first two commits because I assume they're the same as in #6.

File: src/zope/app/onlinehelp/browser/__init__.py (hunk 1 above)
Comment (jamadden) [reply src/zope/app/onlinehelp/browser/__init__.py]: Good catch.

**Comment by jamadden** [comment]
>  the first two commits [are] the same as in #6.
They are, yes. Now that 6 is merged I rebased on master leaving just the one commit here.

Thank you!

**Pull request merged**